fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/CreateRDFSourcePersisterTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.impl;
+
+import org.apache.jena.graph.Triple;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFDataMgr;
+import org.fcrepo.kernel.api.RdfLexicon;
+import org.fcrepo.kernel.api.RdfStream;
+import org.fcrepo.kernel.api.operations.RdfSourceOperation;
+import org.fcrepo.kernel.api.rdf.DefaultRdfStream;
+import org.fcrepo.persistence.ocfl.api.OCFLObjectSession;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.InputStream;
+import java.util.stream.Stream;
+
+import static org.apache.jena.graph.NodeFactory.createLiteral;
+import static org.apache.jena.graph.NodeFactory.createURI;
+import static org.apache.jena.rdf.model.ModelFactory.createDefaultModel;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author dbernstein
+ * @since 6.0.0
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class CreateRDFSourcePersisterTest {
+
+    @Mock
+    private RdfSourceOperation operation;
+
+    @Mock
+    private OCFLObjectSession session;
+
+    @Mock
+    private FedoraOCFLMapping mapping;
+
+    @Captor
+    private ArgumentCaptor<InputStream> userTriplesIsCaptor;
+
+    @Captor
+    private ArgumentCaptor<InputStream> serverTriplesIsCaptor;
+
+    private CreateRDFSourcePersister persister = new CreateRDFSourcePersister();
+
+    @Test
+    public void test() {
+        final String resourceId = "info:fedora/parent/child";
+        final String parentResourceId = "info:fedora/parent";
+
+        //create some test user triples
+        final String title = "my title";
+        final Stream<Triple> userTriples = Stream.of(Triple.create(createURI(resourceId),
+                createURI("http://purl.org/dc/elements/1.1/title"),
```

Comment:
There's a convenient constant for this, `DC.title.asNode()`

Reply:
will do.